ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,295 @@
+  alerts,
+}: {
+  label: string,
+  status: Status,
+  alerts: Alert[],
+}) => {
+  let date = null;
+  const intl = useIntl();
+
+  const getStartDate = (alerts: Alert[]) => {
+    // Check we only have alerts and set startsAt properties
+    const filtered = alerts.filter((item) => item && item.startsAt);
+    // Sort to make sure we have oldest alert first
+    filtered.sort((a, b) => new Date(a.startsAt) - new Date(b.startsAt));
+
+    if (filtered[0] && filtered[0].startsAt) return filtered[0].startsAt;
+    return '';
+  };
+
+  if (alerts.length) date = getStartDate(alerts);
+
+  if (!alerts.length && status === STATUS_HEALTH)
+    return (
+      <ServiceItemElement aria-label={label}>
+        <ServiceItemLabelWrapper>
+          <CircleStatus status={status} />
+          <ServiceItemLabel>{label}</ServiceItemLabel>
+        </ServiceItemLabelWrapper>
+      </ServiceItemElement>
+    );
+  else
+    return (
+      <NonHealthyServiceItemElement>
+        <Tooltip
+          // Right placement to avoid Z index issues between left sidebar and tooltip or out of screen tooltip displays
+          placement="top"
+          overlayStyle={{
+            // em sizing to handle font-size change on large screens
+            width: '20em',
+            height: '100%',
+          }}
+          overlay={
+            <NonHealthyPopUp>
+              <NonHealthyPopUpTitle>
+                {intl.formatMessage({ id: 'view_details' })}
+              </NonHealthyPopUpTitle>
+              <NonHealthyPopUpItem>
+                <label>{intl.formatMessage({ id: 'severity' })}</label>
+                <StatusText status={status}>{status}</StatusText>
+              </NonHealthyPopUpItem>
+              {
+                // Checking date validity
+                date && (
+                  <NonHealthyPopUpItem>
+                    <label>{intl.formatMessage({ id: 'start' })}</label>
+                    <div>{formatDateToMid1(date)}</div>
+                  </NonHealthyPopUpItem>
+                )
+              }
+            </NonHealthyPopUp>
+          }
+        >
+          <Link to="/alerts" data-testid="alert-link">
+            <ClickableServiceItemElement aria-label={label}>
+              <ServiceItemLabelWrapper>
+                <CircleStatus status={status} />
+                <ServiceItemLabel>{label}</ServiceItemLabel>
+              </ServiceItemLabelWrapper>
+              <ClickableIcon className="fas fa-angle-right" />
+            </ClickableServiceItemElement>
+          </Link>
+        </Tooltip>
+      </NonHealthyServiceItemElement>
+    );
+};
+
+const DashboardServices = () => {
+  const intl = useIntl();
+  const alertsLibrary = useAlertLibrary();
+
+  // K8s Master
+  const k8sHighestSeverityAlert = useHighestSeverityAlerts(
+    alertsLibrary.getK8SMasterAlertSelectors(),
+  );
+  const k8sStatus = highestAlertToStatus(k8sHighestSeverityAlert);
+
+  // Bootstrap
+  const bootstrapAlerts = useHighestSeverityAlerts(
```

Comment:
```suggestion
  const bootstrapHighestSeverityAlert = useHighestSeverityAlerts(
```
same for the rest...